GPIO_NUM specifies the GPIO for the interrupt, which is mapped to MIO[76]
GPIO_NUM_CHIP_EN specifies the GPIO for the enable, which is mapped to MIO[8]
GPIO_NUM_RESET specifies the GPIO for the reset_n
GPIO_NUM_RESET can be mapped to either MIO[7] (1.8V I/O) or EMIO[0] (1.2V I/O)
GPIO_NUM_RESET on MIO[7] is the production default
There are 78 MIO GPIOs (0-77) so the first EMIO is #78.
For PetaLinux 2017.4 the Zynq MPSoC GPIO base is 322
The WiFi interrupt is connected to MIO[76], so GPIO_NUM = 322+76 = 398
The WiFi EN is connected to MIO[8], so GPIO_NUM_CHIP_EN = 322+8 = 330
If the WiFi RSTN is mapped to EMIO[0], then GPIO_NUM_RESET = 322+78 = 400
If the WiFi RSTN is mapped to MIO[7], then GPIO_NUM_RESET = 322+7 = 329
diff --git a/wilc1000/wilc_wlan.h b/wilc1000/wilc_wlan.h
index b7cb1b7..5b5186a 100644
--- a/wilc_wlan.h
+++ b/wilc_wlan.h
@@ -144,9 +144,9 @@
 #define LINUX_TX_SIZE		(64 * 1024)
 
 #define MODALIAS		"WILC_SPI"
-#define GPIO_NUM		0x5B
-#define GPIO_NUM_CHIP_EN	94
-#define GPIO_NUM_RESET		60
+#define GPIO_NUM		398
+#define GPIO_NUM_CHIP_EN	330
+#define GPIO_NUM_RESET		329
 
 
 #define NQUEUES			4
